_board/uchip.md

```diff
@@ -47,6 +47,13 @@ You can program uChip using virtually any IDE, in many languages (CircuitPython
 * Standards narrow-DIP footprint: 0.3” (7.62 mm) row spacing, 0.1” (2.54) pin spacing.
 * Pinout standard logic CMOS compatible: power and GND are on pin 16 and 8, so you can also emulate some 16 pin CMOS ICs (4000 and 74HC series)!  
 
+**Additional Notes:**
+
+* The uChip does not have an RGB LED.
+* The on-board LED is amber, rather than green.
+* The board's bootloader is BOSSA-compatible, but does not support UF2.  Use the .bin image of CircuitPython with BOSSA.
+* I've created a Windows .bat script using the BOSSAC commandline tool to flash CircuitPython; see https://github.com/mew-cx/circuitpython_stuff/tree/main/uChip Please customize as needed.
```

Comment:
> The collected details add value.

Indeed, that's the point: _all_ details are captured in a self-documented script; even the filename describes the specifics.  It serves as a single-point-of-truth fully-functional template available for tweaking.  I am my own customer (I use the script myself, tweaking as needed), rather than creating a potentially out-of-sync description for others.

> the binary is in $PATH.

The BOSSA installer does not modify PATH.

> (Sorry, I'm not a Windows users):

Neither am I, really.

Reply:
@mew-cx We have a writeup of using bossac here: https://learn.adafruit.com/welcome-to-circuitpython/non-uf2-installation. I think the main point to make is that you need to use bossac: there is no UF2 bootloader (If I understand correctly). I think it would make sense to refer to the guide page, since it probably subsumes your more specific script.